.github/workflows/image-cuda.yml

```diff
@@ -0,0 +1,109 @@
+name: Test build cuda container image
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'containers/cuda/Containerfile'
+      - '.github/workflows/image-cuda.yml'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'containers/cuda/Containerfile'
+      - '.github/workflows/image-cuda.yml'
+
+# Note that the current containerfile builds against a git ref.
+# It is not built against the current source tree. So, we test
+# build the image against `stable` and `main` if the file changes.
+jobs:
+  build_cuda_image_stable:
+    name: Build CUDA image for stable
+    runs-on: ubuntu-latest
+    steps:
+      - name: Free disk space
```

Comment:
I noticed that GHA have a large, mostly unused `/mnt` mount point with over 65GB of free disk space. Try this hack: https://github.com/tiran/instructlab-containers/blob/cbe5957fd94a375dad3c93c53264cf55a05a6a3a/.github/workflows/containers.yaml#L70-L82

Reply:
nice.

The current changes are enough for these jobs, at least for now.

I also found this github action which should work if we configure it for our use case -- https://github.com/marketplace/actions/maximize-build-disk-space